CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## [1.2](https://github.com/arborx/arborx/tree/v1.2) (2022-03-??)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.1...v1.2)
+
+**New features:**
+
+- Implement minimum spanning tree algorithm [\#624](https://github.com/arborx/ArborX/pull/624) and [\#631](https://github.com/arborx/ArborX/pull/631)
+
+**Build changes:**
+
+- Require Kokkos 3.4 [\#578](https://github.com/arborx/ArborX/pull/578)
+
+**Enhancements:**
+
+- Indicate the status of the worktree in the hash string ("-dirty") [\#558](https://github.com/arborx/ArborX/pull/558)
+- Check valid callback and space accessibility [\#563](https://github.com/arborx/ArborX/pull/563)
+- Add a molecular dynamics example [\#564](https://github.com/arborx/ArborX/pull/564)
+- Add nightly testing against Kokkos develop [\#569](https://github.com/arborx/ArborX/pull/569)
+- Improve performance of the HIP backend when using HIP-Clang [\#575](https://github.com/arborx/ArborX/pull/575)
+- Remove PGI CI [\#590](https://github.com/arborx/ArborX/pull/590)
+- Add Point-Sphere intersection algorithm [\#584](https://github.com/arborx/ArborX/pull/584)
+- Avoid host copy in distributed tree construction when using CUDA-aware MPI [\#597](https://github.com/arborx/ArborX/pull/597)
+- Print Kokkos version during configuration [\#609](https://github.com/arborx/ArborX/pull/609)
+- Improve performance of the brute force algorithm [\#616](https://github.com/arborx/ArborX/pull/616)
+- Use 64-bit Morton indices by default in the construction [\#637](https://github.com/arborx/ArborX/pull/637)
+- Allow alternate space filling curves (EXPERIMENTAL) [\#646](https://github.com/arborx/ArborX/pull/646)
+- Remove certain helper functions from the `ArborX` namespace [\#648](https://github.com/arborx/ArborX/pull/648)
+
+**Fixed bugs:**
+
+- Enable testing when ARBORX_ENABLE_BENCHMARKS is ON [\#555](https://github.com/arborx/ArborX/pull/555)
+- Fix the exit status of the DBSCAN example [\#566](https://github.com/arborx/ArborX/pull/566)
+- Bring DBSCAN README documentation in line with the code [\#568](https://github.com/arborx/ArborX/pull/568)
+- Fix a race condition in the BVH construction [\#579](https://github.com/arborx/ArborX/pull/579)
+- Fix a bug in the nearest search using the DistributedTree [\#653](https://github.com/arborx/ArborX/pull/653)
```

Comment:
and improved performance

Reply:
Not really, we never had any performance in the first place, because it was not working correctly.

Reply:
Lower memory footprint, one less kernel launch

Reply:
Too much info IMHO. Fell free to update.